libs/agentc_core/agentc_core/util/ddl.py

```diff
@@ -68,6 +70,50 @@ def is_index_present(
         return False, e
 
 
+def get_no_of_fts_nodes(conn) -> tuple[int | None, Exception | None]:
```

Comment:
can you add type for conn input parameter? it would make it less confusing while going through the code